ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -424,6 +430,23 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    private func updateSheetExpandedContentHeight() {
+        if let bottomSheetController = bottomSheetController {
+            bottomSheetController.preferredExpandedContentHeight = fittingTableViewHeight + Constants.BottomSheet.expandedContentTopMargin
+        }
+    }
+
+    private var fittingTableViewHeight: CGFloat {
```

Comment:
Probably worth updating the name (and adding a comment) that this is an estimated height? "fitting" in UIKit parlance implies actually measured